tests/connect/test_sort.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+from pyspark.sql.functions import col
+
+
+def test_sort(spark_session):
+    # Create DataFrame from range(10)
+    df = spark_session.range(10)
+
+    # Sort the DataFrame by 'id' column in descending order
+    df_sorted = df.sort(col("id").desc())
+
+    # Verify the DataFrame is sorted correctly
+    df_pandas = df.toPandas()
+    df_sorted_pandas = df_sorted.toPandas()
+    assert df_sorted_pandas["id"].equals(df_pandas["id"].sort_values(ascending=False).reset_index(drop=True)), "Data should be sorted in descending order"
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +6 to +17]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->we'll want to add a bunch more tests for things like sorting by multiple columns, mixing of ascending & descending, default sort order, and so on. 

<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/Eventual-Inc/Daft/3357?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
actually we don't support some of these... I'd prefer making an issue as we have very minimal things implemented yet where it is hard to do multi col stuff

https://github.com/Eventual-Inc/Daft/issues/3424